platforms/android/tangram/src/main/java/com/mapzen/tangram/HttpResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.mapzen.tangram;
+
+import java.io.IOException;
+
+/**
+ * {@code HttpHandler} interface implemented internally in {@link MapController#startUrlRequest(String, long)}
+ * Passes network responses from underlying implementation to be processed internally
+ */
+public interface HttpResponse {
```

Comment:
Yeah `HttpCallback` is better. But...

Since its internally implemented to "bridge" calls to the native network callbacks, but needs to be public to be accessed and called from client httpHandler implementation, how about `HttpCallbackBridge` or something on those lines?